src/webots/gui/WbPreferencesDialog.cpp

```diff
@@ -288,11 +290,21 @@ QWidget *WbPreferencesDialog::createGeneralTab() {
   layout->addWidget(mDisableSaveWarningCheckBox, 6, 1);
 
   // row 7
+  mTelemetryCheckBox = new QCheckBox(tr("Send technical data to Webots developers"), this);
+  mTelemetryCheckBox->setToolTip(tr("We need your help to continue to improve Webots: more information at:\n"
+                                    "https://cyberbotics.com/telemetry"));
```

Comment:
I am wondering if we should not somehow document this rather in the user guide.
This will be more convenient for developers as it will be possible to change the documentation together with the doc.